CHANGELOG.md

```diff
@@ -11,6 +11,9 @@
 - Now requires Swift 4.1 or newer.  
   [Yonas Kolb](https://github.com/yonaskolb)
   [#228](https://github.com/stencilproject/Stencil/pull/228)
+- You can now use brackets in boolean expressions to change operators precedence.  
```

Comment:
Weird that I didn't catch this before. Generally "brackets" (in en-US) is used to refer to "[ ]", whereas "parentheses" is used for "( )". Also pretty sure that in this situation it's "operator" (without an `s`).

> You can now use parentheses in boolean expressions to change operator precedence.